### DISCUSSION TITLE
Cyan 1.2.4

### DISCUSSION THREAD

**Discussion by rodlie** [rodlie/cyan#110] [General]
Fourth patch release for Cyan 1.2.

## Changes (compared to 1.2.3)

 * Fixed Gimp path detection bug (if clean install of Gimp)
 * Fixed macOS support

## Requirements

* Intel Core 2 compatible CPU (SSE4.1)
* 4GB RAM (8+ recommended)
* Windows 7+
* macOS High Sierra 10.13+
* Linux (X11, glibc 2.17+, libgcc 4.8.5+)

<hr /><em>This discussion was created from the release <a href='https://github.com/rodlie/cyan/releases/tag/1.2.4'>Cyan 1.2.4</a>.</em>

**Comment by jules101uk2** [comment]
Hi there, I'm new to using Cyan and I can't get past setting up the default colour profiles at the bottom of the screen. Specifically the drop down labelled 'GRAY' appears to be empty or non responsive.

I'm using a Windows 11 Home.

I've taken the app off and put it back on a few times, but to no avail.

Any help or suggestions, appreciated.

![image](https://user-images.githubusercontent.com/87947023/160207208-17c176a2-05bd-4988-a0e8-29ba6cf187e7.png)

**Reply by rodlie** [reply]
Cyan should install fallback profiles for GRAY, try installing this profile: https://github.com/rodlie/cyan/raw/1.2/res/icc/gray.icc

**Reply by jules101uk2** [reply]
That fixed it, Thank you very much for a prompt reply. :-)

**Comment by chk1xn** [comment]
I've successfully installed CYAN on GIMP 2.10.38 and upon it's first use from within GIMP it started the CYAN GUI and I got prompted to set defaults. Color conversions were easy and straight forward from this point. (eg open an RGB JPG image then go to Cyan > Export Image and I got prompted where to save the CMYK JPG image...)

However, later I've decided to move the CYAN tool to a different location on my Win10. After moving the whole folder I've started CYAN again and quit which updated cyanbin in ~\AppData\Roaming\GIMP\2.10\plug-ins\cyan.py. Then I re-started GIMP but nothing happens now when I do Cyan > Export Image. The file gets marked as dirty but when I export it to JPG, its color profile is still sRGB.

Setting the color profiles back to "empty" ie "Select..." in CYAN and then re-starting GIMP doesn't help either... 

I guess I'm missing something... like how can I completely remove/uninstall CYAN and start from scratch?

**Reply by chk1xn** [reply]
In addition but I'm not sure if it's related, when I try to work with CYAN directly, very often I get this error:
Cyan.exe: Backing store not supported `...path to whatever.jpg' @ error/jpeg.c/JPEGErrorHandler/338
However, on some JPGs (and other formats like PNG) it works. No idea why. And my initial problem (nothing happens when triggering CYAN from GIMP) still exists even with such images that can be opened directly with CYAN...

Edit: Found out this error only shows when trying to open progressive JPEGs.

**Reply by rodlie** [reply]
Probably an issue with libjpeg and how it's built. An alternative could be to offer binaries with other jpeg options (libjpeg-turbo?).

**Reply by chk1xn** [reply]
thanks rodlie. that JPG issue is not really important to me, just noticed it but I have a workaround now.
any idea about my initial problem with CYAN not working any longer after I've moved it to a different location?

**Comment by chk1xn** [comment]
How can I start from scratch with CYAN in GIMP?
After I had installed GIMP for the first time and added CYAN the integration worked nicely.
After I've moved CYAN to a different location it stopped working even after updating cyan.py.
So I'd like to completely start all over. How to do this properly?
Thanks.
